config/src/config/storage_config.rs

```diff
@@ -78,20 +79,23 @@ pub struct StoragePrunerConfig {
     /// store.
     pub ledger_prune_window: Option<u64>,
     /// Batch size of the versions to be sent to the pruner - this is to avoid slowdown due to
-    /// issuing too many DB calls and batch prune instead.
-    pub pruning_batch_size: usize,
+    /// issuing too many DB calls and batch prune instead. For ledger pruner, this means the number of versions to prune a time. For state pruner, this means the number of stale nodes to prune a time.
```

Comment:
nit: split the comment for ledger_store v.s. state store and put them right before the corresponding variables.

Reply:
Done